.github/workflows/tests.yml

```diff
@@ -2,26 +2,30 @@ name: "Testing"
 
 on:
   push:
-    paths-ignore:
-    - 'tutorials/**'
-    - 'README.md'
-    - '.gitignore'
-    - 'examples/**'
+    branches:
+      - main
+  pull_request:
```

Comment:
This will now run the tests after the PR was merged; good failsafe for the production branch (`main`)